projects/ngx-maplibre-gl/src/lib/popup/popup.component.ts

```diff
@@ -11,27 +11,28 @@ import {
   Output,
   SimpleChanges,
   ViewChild,
-} from '@angular/core';
-import { LngLatLike, Offset, Popup, PopupOptions } from 'maplibre-gl';
-import { MapService } from '../map/map.service';
-import { MarkerComponent } from '../marker/marker.component';
+} from "@angular/core";
```

Comment:
Can this changes be reverted? I think the quotes are OK, aren't they?

Reply:
It's done automatically by prettier....
Is there a prettier config file in this repo?

Reply:
Maybe I'll just search and replace all " with ' in this repo?

Reply:
No, that's a bad idea...

Reply:
Fixed it with lint